charts/knative-serving-istio/values.yaml

```diff
@@ -433,3 +440,35 @@ clusterLocalGateway:
           name: http2
         - port: 443
           name: https
+############################################################
+# Monitoring related - Prometheus Service Monitors and Pod Monitors
+############################################################
+monitoring:
+  istioEnvoy:
+    enabled: false
+    namespaceSelector: {}
+    selector:
+      matchExpressions:
+        - key: service.istio.io/canonical-name
+          operator: Exists
+    envoyStats:
+      - port: ".*-envoy-prom"
+        path: /stats/prometheus
+    metricRelabelings:
+      - regex: "(.*_canonical_.*|connection_security_policy|destination_app|destination_cluster|destination_principal|destination_service|destination_service_namespace|destination_version|destination_workload|response_flags|source_app|source_cluster|source_principal|source_version|source_workload|source_workload_namespace)"
+        action: labeldrop
+  serving:
+    # Install Monitoring CRs related to knative service pods
+    enabled: false
+    podMonitor:
+      userMetricPortName: http-usermetric
+      userPortName: user-port
+      selector:
+        matchExpressions:
+          - key: "serving.knative.dev/revision"
+            operator: Exists
+      metricRelabelings:
```

Comment:
I'd make the whole `metricRelabelings` block available for customization in values.yaml. the same is applicable for other monitors

Reply:
like [here](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/values.yaml#L433-L439)

Reply:
understood 👍